site/docs/v1/tech/apis/_tenant-response-body-base.adoc

```diff
@@ -488,6 +506,18 @@ The duration for the number of times a user can request a two-factor code by ema
 [field]#{base_field_name}.registrationConfiguration.blockedDomains# [type]#[Array<String>]# [optional]#Optional# [since]#Available since 1.30.0#::
 A list of unique domains that are not allowed to register when self service is enabled.
 
+[field]#{base_field_name}.scimServerConfiguration.clientEntityTypeId# [type]#[UUID]# [since]#Available since 1.36.0#::
+The Entity Type that will be used to represent SCIM Clients for this tenant.
+
+[field]#{base_field_name}.scimServerConfiguration.enabled# [type]#[Boolean]# [since]#Available since 1.36.0#::
+Whether or not this tenant has the SCIM endpoints enabled.
+
+[field]#{base_field_name}.scimServerConfiguration.schemas# [type]#[Map]# [since]#Available since 1.36.0#::
+JSON formatted as a SCIM Schemas endpoint response. Because the SCIM lambdas may modify the JSON response, ensure the Schema's response matches that generated by the response lambdas.
```

Comment:
Same as above. A link to the SCIM Schemas schema might be helpful and maybe a mention of the default value.

Reply:
Sounds good. I think that the link is helpful, but the default value belongs in the request, not the response.